### DISCUSSION TITLE
Suricata in Disabled state

### DISCUSSION THREAD

**Discussion by heisenberg1977** [Security-Onion-Solutions/securityonion#4204] [Unsupported Versions] [answered]
New to SO 2.3. I installed SO as a VirtualBox VM as an import node to test so-import-pcap. When checking the status I notice that Suricata often goes into disabled state. I can restart it manually but when I check back a few seconds later it is disabled again. Looking for assistance with troubleshooting.

**Comment by heisenberg1977** [comment]
Found in suricata.log:

16/5/2021 -- 21:06:35 - <Error> - [ERRCODE: SC_ERR_AFP_CREATE(190)] - Unable to find iface bond0: No such device
16/5/2021 -- 21:06:35 - <Error> - [ERRCODE: SC_ERR_AFP_CREATE(190)] - Couldn't init AF_PACKET socket, fatal error
16/5/2021 -- 21:06:35 - <Error> - [ERRCODE: SC_ERR_FATAL(171)] - thread W#01-bond0 failed

**Comment by dougburks** [comment, marked as answer]
If you selected Import Mode during installation, then it is normal for the Suricata service to show as disabled since we don't want it sniffing traffic from a network interface. When you run `so-import-pcap`, it then spawns a Suricata instance to read the pcap and Suricata shuts down when finished.

**Comment by heisenberg1977** [comment]
Thanks for the explanation Doug. The reason that I went down the path of troubleshooting Suricata is that when I imported my initial couple of sample PCAPs from Malware Traffic Analysis, I am only seeing entries for Zeek under Hunt - Group Metrics.
